docs/rules/no-unguarded-risky-node-api-in-ssrable-components.md

```diff
@@ -0,0 +1,29 @@
+# Disallow unguarded risky Node API calls in SSR context (`lwc/no-unguarded-risky-node-api-in-ssrable-components`)
```

Comment:
Instead of saying "risky", let's just say unsupported. Technically, the APIs are available, but we won't fix them if a component developer tries to use one and it breaks.

Reply:
Done.